internal/testgen/gen_useless_assert.go

```diff
@@ -71,13 +70,7 @@ func (g UselessAssertTestsGenerator) TemplateData() any {
 		}
 	}
 
-	sort.Slice(twoSideAssertions, func(i, j int) bool {
-		lhs, rhs := twoSideAssertions[i], twoSideAssertions[j]
-		if lhs.Fn == rhs.Fn {
-			return lhs.Argsf < rhs.Argsf
-		}
-		return lhs.Fn < rhs.Fn
-	})
+	sortAssertions(twoSideAssertions)
```

Comment:
let's look we can just leave the sort code here only

because there is `map` above

Reply:
OK, I will remove the others